docs/data-sources/package_install.md

```diff
@@ -97,8 +97,9 @@ Read-Only:
 
 Read-Only:
 
-- `inline_values` (Map of String)
```

Comment:
@ishangupta-ds if it is deprecated, then what is the reason to keep this?
Based on the release version the document will be shown accordingly, right?

Reply:
i will add a deprecated text.

Reply:
Added.